curves/Cargo.toml

```
@@ -69,4 +70,4 @@ version = "0.8"
 rustc_version = "0.4"
 
 [features]
-default = [ "snarkvm-fields/default", "snarkvm-utilities/default" ]
+default = [ ]
```

In general these two features don't have much to do with parallelization after the removal of the `parallel` flags from them. The remaining features have to do with `no-std` environments and byte serialization so, in lieu of rigorous testing for side effects removal of these would cause, these should be left in place

```suggestion
default = [ "snarkvm-fields/default", "snarkvm-utilities/default" ]
```